Sources/Shared/Models/VIMInteraction.m

```diff
@@ -93,4 +89,20 @@ - (void)setStreamStatus
     self.streamStatus = [number intValue];
 }
 
+- (BOOL)canGet {
+    return (self.options && [self.options containsObject:@"GET"]);
```

Comment:
can we use the `HTTPMethod` string values here instead of hard coding our own?

Reply:
Yes and I think you just made a great case for rewriting this in Swift 😂. Right now our HTTPMethod constants are defined in a Swift enum.